woocommerce/woocommerce
# Apply styles to Coming soon entire store regardless of theme

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/team-ghidorah/issues/306.

Update the entire store pattern styles and deregister the styles from the themes or other plugins to ensure the page looks consistent regardless of the theme.

- Added Inter and Cardo font
- Update styles
- Deregister all styles except for block styles.

<img width="1145" alt="Screenshot 2024-04-16 at 15 06 42" src="https://github.com/woocommerce/woocommerce/assets/4344253/c28e928a-90ff-43a9-8d68-467c152bc842">




### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a fresh site with `launch-your-store` feature flag enabled.
2. Go to `/wp-admin/admin.php?page=wc-settings&tab=site-visibility`
3. Make sure `Coming soon` option is checked and `Restrict to store pages only` is off.
4. Visit the site in incognito mode, confirm the coming soon page is displayed properly.
5. Change the theme to a different theme and confirm the coming soon page is displayed properly.
6. Repeat the above steps with different themes



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
